### PR TITLE
init ticket page

### PR DESCRIPTION
**Pull request by TGlide**

### CODE REVIEW

File: .vscode/settings.json
Comment (TorstenDittmann) [inline .vscode/settings.json:1]: can we put .vscode into gitignore :)

File: package.json [hunk 1]
```diff
@@ -56,16 +59,38 @@
     },
     "type": "module",
     "dependencies": {
+        "@appwrite.io/console": "^0.4.2",
         "@appwrite.io/pink": "0.1.0-next.9",
         "@appwrite.io/pink-icons": "0.1.0-next.9",
         "@appwrite.io/repo": "github:appwrite/appwrite#main",
+        "@resvg/resvg-js": "^2.6.0",
         "@splinetool/viewer": "0.9.455",
+        "appwrite": "^13.0.1",
```
Comment (TorstenDittmann) [inline package.json:68]: you should be able to just use `@appwrite.io/console` instead

File: scripts/download-contributor-data.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+const outputFile = `./src/lib/contributors.ts`;
+
+async function fetchContributors() {
+    let page = 1;
+    let contributorsData = [];
+    let hasMoreData = true;
+
+    while (hasMoreData) {
+        console.log(page);
```
Comment (TorstenDittmann) [inline scripts/download-contributor-data.js:14]: is this useful?

File: src/lib/appwrite/init.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { PUBLIC_APPWRITE_PROJECT_INIT_ID } from '$env/static/public';
+import { Client, Account, Databases } from 'appwrite';
```
Comment (TorstenDittmann) [inline src/lib/appwrite/init.ts:2]: here we can use the console sdk too

File: .vscode/settings.json
Comment (TGlide) [reply .vscode/settings.json:1]: Even the custom code snippets? They're useful 😄

File: scripts/download-contributor-data.js (hunk 2 above)
Comment (TGlide) [reply scripts/download-contributor-data.js:14]: The log? Just to keep track of how progress is going.

Comment (TorstenDittmann) [reply scripts/download-contributor-data.js:14]: Can we make the log more meaningful instead of just numbers 😁

File: .vscode/settings.json
Comment (TorstenDittmann) [reply .vscode/settings.json:1]: some people have their own .vscode settings that would conflict 👍🏻

File: package.json (hunk 1 above)
Comment (TorstenDittmann) [reply package.json:70]: this one 👀

**Pull request closed without merging**